install/updates/froxlor/update_2.2.inc.php

```diff
@@ -97,6 +97,10 @@
 	Update::showUpdateStep("Adding new settings");
 	Settings::AddNew("system.le_renew_services", "");
 	Settings::AddNew("system.le_renew_hook", "systemctl restart postfix dovecot proftpd");
+	Settings::AddNew("system.mda_reload_command", "service dovecot reload");
```

Comment:
a new update section with a new db-version is required or users on that version will not get the updates